### ISSUE TITLE
(i2pd.nix) value is a function while a Boolean was expected [20.03]

### ISSUE THREAD

**Issue by curbengh** [NixOS/nixpkgs#86540]
**Describe the bug**
Trying to enable `i2pd` service in 20.03 throws an error (no issue in 19.09):

```
# nixos-rebuild build --show-trace
called from /nix/var/nix/profiles/per-user/root/channels/nixos/nixos/modules/services/networking/i2pd.nix:162:8:
value is a function while a Boolean was expected
```

Points to

https://github.com/NixOS/nixpkgs/commit/9bab9e2ec6be97489ab712b69f7fbb3ece4739d7#diff-37149d49e0c6f771a32bc9053c70ad19R162

**To Reproduce**

``` js
  services.i2pd = {
    enable = true;
  };
```

**Notify maintainers**
@edwtjo

Relates to https://github.com/NixOS/nixpkgs/issues/77189.

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.35-hardened, NixOS, 20.03.1619.ab3adfe1c76 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.4`
 - channels(root): `"nixos-20.03.1619.ab3adfe1c76"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by woffs** [comment]
Just examining: this was broken by 1d61efb7f177f7b70c467ab4940fde0a3481d4dc and fixed by #63103 in master.

**Comment by rnhmjoj** [comment]
This should have been fixed by #88884. If you still have problems after the next channel update, please reopen the issue.

**Comment by curbengh** [comment]
Verified fix in 20.03.2015.e7752db2fb6
